components/panel/src/index.tsx

```diff
@@ -31,7 +31,7 @@ const StyledPanel = styled('div')(
     },
 
     color: WHITE,
-    background: TURQUOISE,
+    background: BUTTON_COLOUR,
```

Comment:
Sorry I never got back on this one. I don't think we should be using BUTTON_COLOUR here.

If you still need this, please can you reference what the issue in the PR description? Is govuk-react out of sync with the design system here? Perhaps add some links or screenshots?

Can we access the colour we need from govuk-colours by another name?